cpp/include/cuspatial/utility/vec_2d.hpp

```diff
@@ -1,14 +1,33 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 #pragma once
 #include <cuspatial/types.hpp>
 
 namespace cuspatial {
 
+/**
+ * @addtogroup type_definition_operator
```

Comment:
What is `type_definition_operator` intended to denote? Why not just `types`? I see that these types are defined in `utility` instead of directly in `types.hpp`, I guess that's because with the header-only API it will be possible for users to never interact directly with these objects? A simpler name like `types` or `coordinate_types` would be more useful.

Reply:
`alignas` is a function that is used to calculate some things for defining aligned types, so `type_definition_operator` seems fine? :shrug: `types` seems conflated and `coordinate_types` seems inappropriate.

Reply:
I'm not sure what you're getting at with the `alignas`. That's not a function defined in this namespace, that's just a specifier for the `vec_2d` type. What is being conflated here?